packages/cli/test/scripts/update.test.js

```diff
@@ -484,6 +530,7 @@ describe('update script', function() {
   const shouldHandleUpdateOnDependency = function() {
     describe('updating on dependency', function() {
       beforeEach('setup', async function() {
+        this.timeout(10000);
```

Comment:
Did you add this in because of the timeouts in CI? Did it fix things? I did not think that the problem was actual lack of time, I was assuming there was something preventing progress.

Reply:
Yes. It was consistently timing out on my laptop. I think it takes a lot of time because it does multiple `push` and `createProxy` commands which take time on Ganache.